source/includes/_gettingstarted.md

```diff
@@ -86,43 +87,20 @@ In Browser JS it is possible to either:
 - Use the standalone functions and pass in a different portal as the first argument.
 - Create a new client with the desired portal. Clients implement all the standalone functions as methods with bound `portalUrl` so you don't need to repeat it every time. See the code example on the right.
 
-### Other Languages
-
-In other languages, a different portal can be passed in as part of the options for each function.
-
 ## Setting Additional Options
 
 Each SDK function also accepts additional options. These vary depending on the endpoint and are documented alongside each function.
 
 <aside class="notice">
-In most of our SDKs the additional options may be left out of the function calls, in which case the default options are used. In Go the options must always be passed in, as the language does not support optional function parameters.
+In most of our SDKs the additional options may be left out of the function calls, in which case the default options are used. In <b>Go</b> the options must always be passed in, as the language does not support optional function parameters.
```

Comment:
Same here. I think it is easy to manage these types of things at the SDK level. When talking about various parameters we can list it as required or optional by SDK. Then it is also right there for the developer working with that specific function. Devs should have to refer back to Getting Started for subtle things like this. Personal preference.

Reply:
Maybe I don't know what you have in mind, but I don't see a way to document this for every function that wouldn't be repetitive and verbose. Go developers should be able to look at an example and see that a default options object is being used, and because there are no optional parameters in Go they should understand that it is required.